src/time.cr

```diff
@@ -560,8 +560,8 @@ struct Time
   # new_year = Time.utc(2019, 1, 1, 0, 0, 0)
   # tokyo = new_year.to_local_in(Time::Location.load("Asia/Tokyo"))
   # new_york = new_year.to_local_in(Time::Location.load("America/New_York"))
-  # tokyo.to_s    # => 2019-01-01 00:00:00.0 +09:00 Asia/Tokyo
-  # new_york.to_s # => 2019-01-01 00:00:00.0 -05:00 America/New_York
+  # tokyo.to_s    # => "2019-01-01 00:00:00 +09:00"
+  # new_york.to_s # => "2019-01-01 00:00:00 -05:00"
```

Comment:
These examples should use `inspect` instead of `to_s`, which keeps the intended output including the location name.